bundler/man/bundle-gem.1

```diff
@@ -65,7 +65,7 @@ Do not create a \fBLICENSE\.txt\fR (overrides \fB\-\-mit\fR specified in the glo
 .
 .TP
 \fB\-t\fR, \fB\-\-test=minitest\fR, \fB\-\-test=rspec\fR, \fB\-\-test=test\-unit\fR
-Specify the test framework that Bundler should use when generating the project\. Acceptable values are \fBminitest\fR, \fBrspec\fR and \fBtest\-unit\fR\. The \fBGEM_NAME\.gemspec\fR will be configured and a skeleton test/spec directory will be created based on this option\. If this option is unspecified, an interactive prompt will be displayed and the answer will be saved in Bundler\'s global config for future \fBbundle gem\fR use\. If no option is specified, the default testing framework is RSpec\.
+Specify the test framework that Bundler should use when generating the project\.  Acceptable values are \fBminitest\fR, \fBrspec\fR and \fBtest\-unit\fR\. The \fBGEM_NAME\.gemspec\fR will be configured and a skeleton test/spec directory will be created based on this option\. If this option is unspecified, an interactive prompt will be displayed and the answer will be saved in Bundler\'s global config for future \fBbundle gem\fR use\. If no option is specified, this defaults to the global config setting\.
 .
```

Comment:
Maybe be more clear about the complete functionality here, what you think of something like:

`this defaults to the global config setting or in case that nothing is configured it will prompt for a test framework and configure it`

Reply:
This might slightly change depending on: https://github.com/rubygems/rubygems/pull/3544#issuecomment-630700501.

Reply:
I updated the man pages, tried to keep similar language with the rest of the files